.eslintrc

```diff
@@ -113,6 +113,9 @@
         "mocha": true,
         "node": true
     },
+    "globals": {
```

Comment:
This is not necessary. 

It's checked here: https://github.com/HabitRPG/habitrpg/blob/5304b37102f7210b00f63fdefe4041f8ec08a720/tasks/gulp-eslint.js#L61-L65

Going to move these rules to their own `.eslintrc` in the tests directory soon so a normal `eslint path/to/file` will work correctly.